static/app/components/searchQueryBuilder/index.tsx

```diff
@@ -76,6 +76,12 @@ export interface SearchQueryBuilderProps {
    * Sections and filter keys are displayed in the order they are provided.
    */
   filterKeySections?: FilterKeySection[];
+  /**
+   * A mapping of secondary aliases for filter keys.
+   * These are used to ensure that the filter key does not show them as invalid, however
+   * they will not be shown in the filter key dropdown.
+   */
+  filterKeysSecondaryAliases?: TagCollection;
```

Comment:
Hmm referring to these as aliases makes it sound like we are renaming keys. I wonder if there is a better name for this prop 🤔